core/new-gui/src/app/workspace/component/menu/menu.component.scss

```diff
@@ -6,9 +6,6 @@
 #logo {
   display: inline-block;
   vertical-align: top;
-  img {
-    height: calc(var(--header-height) - 1px);
-  }
```

Comment:
it seems to be unrelated to this PR?

Reply:
This PR relaxed the relationship between complements such that this CSS is no longer needed.